test/e2e_node/util.go

```diff
@@ -277,7 +282,7 @@ func logKubeletLatencyMetrics(ctx context.Context, metricNames ...string) {
 	for _, key := range metricNames {
 		metricSet.Insert(kubeletmetrics.KubeletSubsystem + "_" + key)
 	}
-	metric, err := e2emetrics.GrabKubeletMetricsWithoutProxy(ctx, fmt.Sprintf("%s:%d", framework.TestContext.NodeName, ports.KubeletReadOnlyPort), "/metrics")
```

Comment:
does the healthz port serve metrics??

Reply:
My fault.

For metrics, it is similar to `/stats/summary`. 
- https + 10250 
- or HTTP + 10255

I kept this and marked `GrabKubeletMetricsWithoutProxy` as deprecated. We should alter to use `10250`.